data/en/census_household_gb_eng.json

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
I'm guessing this wasn't meant to be committed?

Reply:
It was, it's used to provide quick-launch. I mentioned it in the PR description but will delete once approved.

Reply:
Or now, since Donna is happy.